### PR TITLE
Jenkins Plugins and Base-Tech Update

### PR DESCRIPTION
**Pull request by kaiwalyajoshi**
- Update base-tech to 2.190.1
- Upadte libmesos-bundle to 1.14-beta
- Update mesos-plugin to version 1.0.0
- Update prometheus plugin to 2.0.6
- Update BlueOcean to 1.19.0
- Update all the other bundled plugins to the latest versions.
- Remove `mesos` label for jenkins-agents for scale-test workloads.
- Set `dind=true` for the mesos-plugin by default.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,10 +11,10 @@ ENV JENKINS_FOLDER /usr/share/jenkins
 
 # Build Args
 ARG LIBMESOS_DOWNLOAD_URL=https://downloads.mesosphere.io/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz
```
Comment (kvish) [inline Dockerfile]: bump libmesos to 1.4

Comment (kaiwalyajoshi) [reply Dockerfile]: Good catch, following up with the team to see which version we can ship.

Comment (vinodkone) [reply Dockerfile]: let's use 1.14.0-beta for now.

File: tests/scale/test_load.py [hunk 2]
```diff
@@ -554,14 +556,14 @@ def _create_executor_configuration(service_name: str) -> str:
     Returns: Random name of the new config created.
 
     """
-    mesos_label = "mesos"
+    mesos_label = ""
```
Comment (vinodkone) [inline tests/scale/test_load.py]: why this change?

Comment (kaiwalyajoshi) [reply tests/scale/test_load.py]: affects MWT load only, currently mwt job runs look for agents with a `mesos` label, out-of-the-box the mesos plugin currently doesn't apply any labels.
